CMake/QtDCMConfig.cmake.in

```diff
@@ -2,51 +2,34 @@
 # This file is to be included from outside projects which uses QtM3d
 
 #path to the source of QTDCM
-SET(QTDCM_SOURCE_DIR "@QTDCM_SOURCE_DIR@/Code")
+set(QTDCM_SOURCE_DIR "@QtDcm_SOURCE_DIR@/Code")
 
 #path to cmake files of Vistal
-SET(QTDCM_CMAKE_DIR "@QTDCM_CMAKE_DIR@")
+set(QTDCM_CMAKE_DIR "@QtDcm_SOURCE_DIR@/CMake")
 
 #path to binary dir of QTDCM
-SET(QTDCM_BINARY_DIR "@QTDCM_BIN_DIR@")
+set(QTDCM_BINARY_DIR "@QtDcm_BINARY_DIR@/lib")
 
```

Comment:
My mistake here :S. I let you correct (too lazy to make a pull request just for that, and i can't push on your repo ;) )